cloudbuild.yaml

```diff
@@ -12,3 +12,34 @@ steps:
         fi
     env:
       - "TAG_NAME=${TAG_NAME}"
+
+  - id: deploy-csharp
+    name: "bitnami/dotnet-sdk"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        if [ "$TAG_NAME"  ] ; then
+          cp vmlapis.csproj gen/csharp
+          cd gen/csharp
+          dotnet pack -o . -c Release
+          dotnet nuget push vmlapis.$TAG_NAME.nupkg --source https://api.nuget.org/v3/index.json --api-key $$NUGET_API_KEY
```

Comment:
perhaps we could try to build the packages after every branch push and only publish them on new tag ? to know they can be built before making a release

Reply:
Yeah that sounds good